google-cloud-redis-v1/lib/google/cloud/redis/v1/cloud_redis/client.rb

```diff
@@ -0,0 +1,1005 @@
+# frozen_string_literal: true
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Auto-generated by gapic-generator-ruby. DO NOT EDIT!
+
+require "gapic/common"
+require "gapic/config"
+require "gapic/config/method"
+
+require "google/cloud/errors"
+require "google/cloud/redis/v1/version"
+require "google/cloud/redis/v1/cloud_redis_pb"
+require "google/cloud/redis/v1/cloud_redis/credentials"
+require "google/cloud/redis/v1/cloud_redis/paths"
+require "google/cloud/redis/v1/cloud_redis/operations"
+
+module Google
+  module Cloud
+    module Redis
+      module V1
+        module CloudRedis
+          ##
+          # Client for the CloudRedis service.
+          #
+          # Configures and manages Cloud Memorystore for Redis instances
```

Comment:
Shouldn't this sentence end with a period(`.`)?

Reply:
Yeah ideally, but there is currently no period in the proto source: https://github.com/googleapis/googleapis/blob/master/google/cloud/redis/v1/cloud_redis.proto#L32 so we need to make do.